### DISCUSSION TITLE
Angular Micro Frontends with CI/CD deployment to AWS S3

### DISCUSSION THREAD

**Discussion by markjaysongonzaga** [nrwl/nx#23404] [Feature Requests]
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [x] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?
[CI/CD](https://nx.dev/ci/intro/ci-with-nx)

### Additional context or description
I've been looking for the right resources to deploy micro frontends angular to aws s3. But could not find any. I would like to ask if there is a way to deploy each angular projects to s3 during ci/cd?

Thanks

**Comment by mehrad-rafigh** [comment]
@markjaysongonzaga I think it would be better to move this to the [Discussions](https://github.com/nrwl/nx/discussions)

**Comment by Coly010** [comment]
It should most certainly be possible to deploy to s3 during CI/CD. However, as this tends to be something that is very unique to organisations we do not, at least for now, have any docs on how to do it.

You should be able to follow any normal documentation around deploying SPAs to S3.
A quick google search uncovered this article which might be helpful: https://medium.com/taptuit/deploy-a-single-page-application-spa-to-aws-b9c0c9587917
